travis.sh

```diff
@@ -4,6 +4,11 @@ set -ev
 
 ./liftsh test
 
+cd web/webkit
+npm run lint
+npm run test
+cd -
```

Comment:
We've had this issue before at Elemica—if `./liftsh test` fails but `npm run test` passes, will this properly mark the build as failed?